### PR TITLE
Hwigwang Lim branch

### PR DESCRIPTION
**Pull request by hwigwanglim**
Adding my profile to the group website.
Need to check if it works well; I could not solve issues with "bundle exec jekyll serve".
I could make html files at "_site" directory, so I opened /website/_site/members/hwigwang_lim.html but the first extra photo didn't appear there. Two extra photos total!

### CODE REVIEW

**Review by cbefan — APPROVED** [review]
Web page displays correctly, everything seems in order. Looks great!

**Review by arthur-lin1027 — APPROVED** [review]
things look good and are rendered good!

File: members/hwigwang_lim.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: Graduate Student Researcher
+
+<!-- Name at least one research topic amongst this list -->
+**Things I'm excited about right now**: Behavior and role of particles in ionic liquid. Representation of charged molecules and particles.
+
+<!-- Ultimately, we'll use this section to
+     include papers and talks, and contributions
+     But for now put whatever you want -->
+**Cool stuff I've done**: Carving Bucky Badger on a pumkin, watching movies for 8 hours a day, and all the things meaningless and meaningful at the same time.
```
Comment (rosecers) [inline members/hwigwang_lim.md:20]: ### 
```suggestion
**Cool stuff I've done**: Carved Bucky Badger on a pumpkin, watched movies for 8 hours in one day, and many things meaningless and meaningful at the same time.
```

Comment (hwigwanglim) [reply members/hwigwang_lim.md]: Great! I just added a commit for this suggestion.

**Review by rosecers — APPROVED** [review]
Looks good to me!

**Pull request merged**